autopilot/contractor.go

```diff
@@ -0,0 +1,536 @@
+package autopilot
+
+import (
+	"errors"
+	"math"
+	"sync"
+	"time"
+
+	"go.sia.tech/renterd/bus"
+	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
+	"go.sia.tech/renterd/worker"
+	"go.sia.tech/siad/types"
+)
+
+const (
+	// estimatedFileContractTransactionSetSize is the estimated blockchain size
+	// of a transaction set between a renter and a host that contains a file
+	// contract.
+	//
+	// TODO: move this constant to a separate package along with all others
+	estimatedFileContractTransactionSetSize = 2048
```

Comment:
let's not define a `constants` package unless we *really* need to 😬

Reply:
Oh yeah that `TODO` is terribly phrased, with "all others" I was aiming at the constants we define for estimated sizes of things that concern the wallet. We copy some from `walletd` for instance as well for outputs size. Chris made a comment on another PR saying those should be consolidated in one place.

I personally don't really mind it and would just leave it until it becomes an issue. I just added the TODO because I resolved a comment and I generally never resolve comments without either tackling it or adding a TODO for it, or have the reviewer resolve it himself if it becomes moot.